src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const payment = 40 * days;
+
+  if (days < 3) {
+    return payment;
+  }
+
+  if (days >= 3 && days <= 6) {
```

Comment:
```suggestion
  if (days <=6) {
```
you already checked, that days not less than 3